torch/testing/asserts.py

```diff
@@ -0,0 +1,186 @@
+import functools
+import inspect
+import warnings
+from typing import Callable, Optional, Tuple
+
+import torch
+from torch.testing._internal.common_utils import get_comparison_dtype as _get_comparison_dtype, TestCase as _TestCase
+
+__all__ = ["assert_tensors_equal", "assert_tensors_allclose"]
+
+try:
+    import pytest
+
+    UsageError = pytest.UsageError
+except ImportError:
+
+    class UsageError(Exception):
+        pass
+
+
+class UsageWarning(Warning):
+    pass
+
+
+def _get_default_rtol_and_atol(tensor1: torch.Tensor, tensor2: torch.Tensor) -> Tuple[float, float]:
```

Comment:
This is an internal function but typically when we work with multiple tensors it's "a, b, ..." and when using a single tensor it's "t"